clusterman/draining/queue.py

```diff
@@ -291,7 +292,11 @@ def process_warning_queue(self) -> None:
 
 
 def host_from_instance_id(sender: str, receipt_handle: str, instance_id: str,) -> Optional[Host]:
-    instance_data = ec2_describe_instances(instance_ids=[instance_id])
+    try:
+        instance_data = ec2_describe_instances(instance_ids=[instance_id])
+    except botocore.exceptions.ClientError as e:
+        logger.warning(f"Couldn't describe instance: {e}")
```

Comment:
could also use `logger.exception` and python will automatically include the traceback after logging the message :)